src/routers/openml/datasets.py

```
@@ -442,3 +478,27 @@ def get_dataset(
         collection_date=dataset.collection_date,
         md5_checksum=dataset_file.md5_hash,
     )
+
+
+def tag(data_id: int, tag: str, user_id: int, connection: Connection) -> None:
```

Ruff (0.15.1)

[warning] Unused function argument: `user_id`

(ARG001)